packages/client/src/CollectionItem.ts

```diff
@@ -25,6 +25,15 @@ export class CollectionItem implements UploadableCollectionItem {
             throw new Error("Terms and conditions must include at most one logion classification element");
         }
 
+        const creativeCommons = args.clientItem.termsAndConditions
+            .filter(element => element.type === "CC4.0")
+            .map(element => element as CreativeCommons);
+        if(creativeCommons.length > 0) {
+            this._creativeCommons = creativeCommons[0];
+        } else if(creativeCommons.length > 1) {
+            throw new Error("Terms and conditions must include at most one Creative Commons element");
```

Comment:
So an item may have at the same time a logion classification and CC terms and conditions?

Reply:
Good question. I'll ask our PO. But note the location you point in your comment is not the right place to enforce that rule. [Here](https://github.com/logion-network/logion-api/blob/feature/creative-commons/packages/client/src/LocClient.ts#L605) in LocClient seems better.

Reply:
Indeed, another possibility is to create a VO (e.g. `TermsAndConditions`) enforcing these constraints and passed to `addCollectionItem`.

Reply:
Done in https://github.com/logion-network/logion-api/pull/67/commits/4168cd515f8dc073cd6fd15bd7f40955d43dc607